Aquality.Selenium.Template/Aquality.Selenium.Template/Configuration/Environment.cs

```diff
@@ -0,0 +1,19 @@
+﻿using Aquality.Selenium.Core.Configurations;
+using Aquality.Selenium.Core.Utilities;
+using System.Reflection;
+
+namespace Aquality.Selenium.Template.Configuration
+{
+    internal static class Environment
+    {
+        public static ISettingsFile CurrentEnvironment
+        {
+            get
+            {
+                var envName = EnvironmentConfiguration.GetVariable("environment") ?? "Stage";
```

Comment:
do not we want to move default value into settings.json?

Reply:
EnvironmentConfiguration.GetVariable doesn't read value from settings file